redash/README.md

```
@@ -0,0 +1,63 @@
+
+(Brief description of Redash and its key features)
+
+## Prerequisites
+
+- Docker (version X.X or higher)
+- Docker Compose (version X.X or higher)
+
+## How to run
+
+- init .env
+```
```

Markdownlint

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)